fhir/src/services/observation/observation.service.test.js

```diff
@@ -44,16 +44,19 @@ describe('Observation Service Test', () => {
 
 	describe('Method: getObservation', () => {
 
-		test('should correctly return a document', async () => {
+		test('should correctly return all laboratory documents for this patient', async () => {
 			let query = { patient: 1, category: 'laboratory' };
-			let [ err, doc ] = await asyncHandler(
+			let [ err, docs ] = await asyncHandler(
 				observationService.getObservation({ query }, logger)
```

Comment:
Should we rename the function to be plural?

Reply:
We could but it's kind of a very low priority item for me since this needs to be submitted soon. It would require multiple changes in both repos, including documentation updates. I would like to wait til after standup and see what other tasks are remaining.